integration-tests/tests/src/tests/sync/app.ts

```diff
@@ -235,6 +236,50 @@ describe("App", () => {
       expect(this.app.currentUser).to.be.null;
     });
 
+    it("currentUser is available from an App listener", async function (this: Mocha.Context & AppContext & RealmContext) {
+      expect(this.app.currentUser).to.be.null;
+
+      const credentials = Realm.Credentials.anonymous();
+
+      const handle = createPromiseHandle();
+      const listener = () => {
```

Comment:
Will this function be called once or twice?

Reply:
I expect it to be called only once. It doesn't behave as the collection listeners (where it fires initially).